src/sass/components/_path.scss

```diff
@@ -0,0 +1,82 @@
+// Path Overrides
+// ==============
+
+// redefine path-specific color variables
+$color-text-path-current: $color-text-default;
+$color-background-path-active: $brand-background-primary;
+$color-background-path-active-hover: $color-brand-darker;
+$color-background-path-complete: $color-background-path-incomplete-hover;
+$color-background-path-complete-hover: $color-gray-6;
+
+.slds-path__item {
+  min-width: 3rem;
+  transition: flex 200ms;
+
+  &:hover {
+    flex-grow: 1.5;
+  }
+}
+
+.slds-path__nav {
+  flex: auto;
+
+  .slds-is-active:not(.slds-is-won) {
+    background: $color-background-path-active;
+
+    &:before,
+    &:after {
+      background: $color-background-path-active;
+    }
+
+    &:hover {
+      background: $color-background-path-active-hover;
+
+      &:before,
+      &:after {
+        background: $color-background-path-active-hover;
+      }
+
+      .slds-path__link {
+        color: white;
+      }
+    }
+
+    .slds-path__link {
+      color: $color-text-path-current;
+
+      &:hover {
+        color: white;
+      }
+    }
+  }
+
+  .slds-is-complete:not(.slds-is-won) {
+    background: $color-background-path-complete;
+    flex-grow: 0.5;
+
+    &:before,
+    &:after {
+      background: $color-background-path-complete;
+    }
+
+    &:hover {
+      background: $color-background-path-complete-hover;
+      flex-grow: 1.5;
+
+      &:before,
+      &:after {
+        background: $color-background-path-complete-hover;
+      }
+    }
+
+    .slds-path__link {
+      color: $color-text-default;
+    }
+  }
+
+  .slds-is-won {
+    .slds-path__link {
+      color: $color-text-brand-primary;
+    }
+  }
```

Comment:
This looks redundant to me? But doesn't hurt.

Reply:
when I remove it, the word "merged" is a link blue color for me. Am I not adding another class that may be there that I am not thinking of to test? I am fixing the other "white" color name to the variable

Reply:
@stacyk Ah, I think I see. You must be looking at a step that is `.slds-is-won`, but _not_ `.slds-is-active .slds-is-current`. I don't think that's actually possible in practice (unless I'm missing something?). In fact, the same "blue text" exists in native SLDS, if you remove `.slds-is-active .slds-is-current` from the `.slds-is-won` element: https://www.lightningdesignsystem.com/components/path/#Won

But this doesn't hurt anything either -- I'm fine leaving it in.

Reply:
ah, yes, I tested `slds-is-won` and `slds-is-won slds-is-complete` (or completed, can't remember off the top of my head but that doesn't matter ATM). I didn't think about using active/current